content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -79,6 +79,18 @@ export class _CollapsibleSection extends React.PureComponent {
     }));
   }
 
+  _getSectionBodyHeight() {
+    const div = this.sectionBody;
+    if (div.style.display === "none") {
+      // If the div isn't displayed, we can't get it's height. So we display it
+      // with a 0px height to get the height. We don't undo this because we are about
+      // to expand the section anyway;
+      div.style.maxHeight = "0px";
```

Comment:
@piatra found we can remove this line because we have this already set in the default css -> https://github.com/mozilla/activity-stream/blob/4104d533b8a46a85ab6d1d5756e368a4dfbcbc2f/content-src/components/CollapsibleSection/_CollapsibleSection.scss#L132